ArduCopter/defines.h

```diff
@@ -144,6 +144,7 @@ enum LoggingParameters {
 #define MASK_LOG_IMU_FAST               (1UL<<18)
 #define MASK_LOG_IMU_RAW                (1UL<<19)
 #define MASK_LOG_VIDEO_STABILISATION    (1UL<<20)
+#define MASK_LOG_FTN_FAST               (1UL<<21)
```

Comment:
Shouldn't this be MASK_LOG_FHN_FAST?  "H" for harmonic?  I don't really mind too much (which is why I've approved it).

Reply:
The log message is "FTN" for "Filter Tuning" - I thought it made more sense to align them

Reply:
ok cool